todos/week3/4_classestodo.py

```diff
@@ -15,14 +17,16 @@
 # TODO: of Employee set equal to a variable employeeOne
 
 
-
+# TODO: ###########################################################################################
+# TODO: Section 3:
 # TODO: Set up a child class that derives from the Employee class and add the attribute 'h_benefits'.
 # TODO: Define a function 'intro' to print:
 # TODO: "Hi, my name is Garrett Palmeri. I am 24 year old Financial Analyst that makes $40000 per year.
 # TODO: I do not have insurance."
 
 
-
+# TODO: ############################################################################################
```

Comment:
I do not think that section 4 is necessary as it provides no real value. It is also simpler than the previous problem, so it sort of backtracks the lesson.